File: problemas/1100/1180.cs
using System;
using System.Linq;

namespace uri1180
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());
            var x = new int[n];
            string[] entrada = Console.ReadLine().Split(' ');

            for (int i = 0; i < n; i++)
            {
                x[i] = int.Parse(entrada[i]);
            }

            Console.WriteLine($"Menor valor: {x.Min()}");
            Console.WriteLine($"Posicao: {Array.IndexOf(x, x.Min())}");
        }
    }
}

File: problemas/1100/1180.dart
import 'dart:io';
import 'dart:math';

void main() {
  final n = int.parse(stdin.readLineSync());
  final x = new List<int>();
  final entrada = stdin.readLineSync().split(' ');

  for (var i = 0; i < n; i++) {
    x.add(int.parse(entrada[i]));
  }

  stdout.writeln('Menor valor: ${x.reduce(min)}');
  stdout.writeln('Posicao: ${x.indexOf(x.reduce(min))}');
}